client/src/components/AddClimbedBoulderForm.spec.jsx

```diff
@@ -29,7 +29,7 @@ describe('AddClimbedBoulderForm', () => {
     expect(submitButton).toBeInTheDocument();
   });
 
-  it('submits form data if none of the input fields is filled out', () => {
+  it('submits form data with the prefilled values', () => {
```

Comment:
Well, you changed the description, but you do not test whether the prefilled values are submitted, but you still test whether the `handleAdd` was called.
[Have a look in this example](https://codesandbox.io/s/component-testing-exercise-3-solution-gl7012?file=/src/components/GameForm.spec.js) how to test the submit function and its submitted values.